Custom command support for web interface

Abyss777 opened the pull request:
1. Added TYPE_CUSTOM to list of supported commands for all protocols
2. Extended Web UI

tananaev commented:
Looks good. The only thing is that Traccar uses Transifex for localization, so you need to add English string first, and then when it's merged go to the Transifex platform and do a translation there:

https://www.transifex.com/traccar/traccar

Abyss777 commented:
I've removed Russian string and added more English strings that autogenerated by https://github.com/tananaev/traccar/blob/master/web/app/store/CommandTypes.js#L38

I've tried to repeat your style e.g. all words capitalized. Added verb "set" for `sosNumber` and `silenceTime` and I doubt about verb for `voiceMessage`

tananaev commented:
Merged. Thanks for the pull request.
